### ISSUE TITLE
error in sutime.py

### ISSUE THREAD

**Issue by agrawal-b** [FraBle/python-sutime#40]
In sutime.py, there are certain jar files which are marked as required, and in jars those files have been updated and therefore now sutime.py is showing error, so to resolve this, i manually changed the names of those 'required jar files' in sutime.py.
can you please fix this issue?

**Comment by Utkichaps** [comment]
Yeah for some reason the sutime.py file in the repo and the one after doing pip install don't match even though they are the same version. The code in the repo has the correct version of the java files

**Comment by Utkichaps** [comment]
#41

**Comment by kaushikacharya** [comment]
> Yeah for some reason the sutime.py file in the repo and the one after doing pip install don't match even though they are the same version. The code in the repo has the correct version of the java files

Version of java dependencies have been changed in this commit on 27th Oct 2018: https://github.com/FraBle/python-sutime/commit/67c47980932f1b32e02bd50ae071deaf003e7b4d

https://pypi.org/project/sutime/#history
pip install version=0.2.0 is from Jan 11, 2017
(Though there have been few more releases, but the default version is 0.2.0)

Hence when we download dependencies using the updated pom.xml, there are mismatches regarding dependency jars.
